build/deploy/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM busybox
+FROM buildpack-deps:jessie-curl
```

Comment:
Is there any rationale behind this selection? (I.e. Is this recommended somewhere?)